.github/workflows/test-semgrep-pro-latest.yaml

```diff
@@ -13,6 +13,7 @@ on:
 jobs:
```

Comment:
```suggestion
  push:
    branches:
      - "develop"
jobs:
```

Reply:
Also haven't tested this explicitly, so @tpetr may need to run another test from his fork

Reply:
i see no reason not to do this, so i'm down